presto-verifier/src/main/java/com/facebook/presto/verifier/Verifier.java

```diff
@@ -215,12 +227,12 @@ private VerifierQueryEvent buildEvent(Validator validator)
                 queryPair.getTest().getCatalog(),
                 queryPair.getTest().getSchema(),
                 queryPair.getTest().getQuery(),
-                optionalDurationToSeconds(test.getCpuTime()),
+                checkCpu ? optionalDurationToSeconds(test.getCpuTime()) : null,
```

Comment:
If either of the cpu time filed is `null`, the `HumanReadableEventClient` won't do the comparison.